pkg/kcp/nuke/deleteResources.go

```diff
@@ -3,6 +3,7 @@ package nuke
 import (
 	"context"
 	"github.com/kyma-project/cloud-manager/pkg/composed"
+	. "github.com/kyma-project/cloud-manager/pkg/kcp/nuke/types"
```

Comment:
This is confusing, please use `nuketypes.State` so it's clear when reading the code it's not the `State` from this package